examples/decorator/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "decorator"
+version = "0.1.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+name = "decorator"
+crate-type = ["cdylib"]
+
+[dependencies.pyo3]
+# If you copy this example, you should uncomment this...
+# version = "0.15.1"
+
+# ...and delete this path
+path = "../.."
+features = ["auto-initialize", "extension-module"]
```

Comment:
Hmmm baffling; the CI failure makes it look like this line is not being picked up. The build script shouldn't be emitting a link instruction...